contracts/modules/licensing/LicensingModule.sol

```diff
@@ -331,6 +331,14 @@ contract LicensingModule is
         uint32[] memory rPercents = new uint32[](parentIpIds.length);
         for (uint256 i = 0; i < parentIpIds.length; i++) {
             (address royaltyPolicy, uint32 royaltyPercent, , ) = lct.getRoyaltyPolicy(licenseTermsIds[i]);
+            Licensing.LicensingConfig memory lsc = LICENSE_REGISTRY.getLicensingConfig(
```

Comment:
do we also need this in other callpaths of derivation? Or just in the callpath of `registerDerivativeWithLicenseTokens` ?

Reply:
Yes, we do need this logic in `mintLicenseTokens` and `registerDerivative` code path which has handled in the `_payMintingFee()` function at LN 588 - 561